docs/snippets/quickstart-install.md

```diff
@@ -1,22 +1,60 @@
+!!! todo "Installing the `quickstart` plugin"
+    === "Using Homebrew"
+        For macOS, you can install the `quickstart` plugin by using <a href="https://github.com/knative/homebrew-client">Homebrew</a>.
+            ```
+            brew install knative-sandbox/kn-plugins/quickstart
+            mkdir -p ~/.config/kn/plugins && cp /usr/local/bin/kn-quickstart ~/.config/kn/plugins
```

Comment:
I don't understand why you'd should still have to do this _after_ the brew install

Reply:
Right now, as I understand it brew installs to `/usr/local/bin`, which is not a location that the kn client recognizes as a plugin location. 

We had some discussion [in Slack](https://knative.slack.com/archives/CE4MVFVAQ/p1626966362140300?thread_ts=1626964104.136700&cid=CE4MVFVAQ) about updating the install location, so perhaps we remove the second line.

Reply:
Yeah, I'd say _personally_ I think until we have that change I don't buy that this is better or less scary/off-putting than "curl install.konk.dev" was, so we should wait for that to land before landing this, imo anyway (we'll get there eventually!)

Reply:
Sounds good. I opened a [PR](https://github.com/knative-sandbox/homebrew-kn-plugins/pull/24) with what I think should fix the install location.

Reply:
Do we need to remove this line now, so that we can use the new path lookup stuff instead?